README.md

```diff
@@ -92,15 +92,15 @@ method     | url                                                    | file path
 
 ### Nested Resources
 
-Nesting of one layer at most is supported.
+Nesting of two layer at most is supported.
 
 method     | url                                                                       | file path                         | controller name
 ---        | ---                                                                       | ---                               | ---
-**GET**    | `/doc/api/{parents}/:parent_id/{objects}?per_page={per_page}&page={page}` | `app/apis/{parents}/{objects}.js` | **index()**
-**GET**    | `/doc/api/{parents}/:parent_id/{objects}/:id`                             | `app/apis/{parents}/{objects}.js` | **show()**
-**POST**   | `/doc/api/{parents}/:parent_id/{objects}`                                 | `app/apis/{parents}/{objects}.js` | **create()**
-**PUT**    | `/doc/api/{parents}/:parent_id/{objects}/:id`                             | `app/apis/{parents}/{objects}.js` | **update()**
-**DELETE** | `/doc/api/{parents}/:parent_id/{objects}/:id[s]`                          | `app/apis/{parents}/{objects}.js` | **destroy()**
+**GET**    | `/doc/api/{parents}/:parent_id/{children}/:child_id/{objects}?per_page={per_page}&page={page}` | `app/apis/{parents}/{objects}.js` | **index()**
```

Comment:
If you see any breaking existing behavior, I need to rework on the PR. That is not intention at all.